typhos/utils.py

```diff
@@ -536,13 +533,10 @@ def find_file_in_paths(filename, *, paths=None):
 
         filename = filename.name
 
-    if len(paths) > 1:
-        paths = remove_duplicate_items(
-            [pathlib.Path(p) for p in paths]
-        )
-
     from .cache import _CachedPath
-    paths = [_CachedPath.from_path(p) for p in paths]
+    paths = remove_duplicate_items(
+        [_CachedPath.from_path(p) for p in paths]
+    )
```

Comment:
I believe this to be the same case as above in which we already pass the paths as `paths = cache.get_global_display_path_cache().paths` but I am not sure if that will always be true.